autocone_colin:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  

  # Velocity Controllers ---------------------------------------
  rear_left_axle_ctl:
    type: effort_controllers/JointVelocityController
    joint: base_link_to_rl_wheel
    pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

  rear_right_axle_ctl:
    type: effort_controllers/JointVelocityController
    joint: base_link_to_rr_wheel
    pid: {p: 1.0, i: 0.0, d: 0.0, i_clamp: 0.0}

  #pid: {p: 1.5, i: 0.2, d: 0.0, i_clamp: 10.0}


  # Position Controllers ---------------------------------------
  #left_steering_hinge_position_controller:
  #  joint: left_steering_hinge_joint
  #  type: effort_controllers/JointPositionController
  #  pid: {p: 1.0, i: 0.0, d: 0.5}

  #right_steering_hinge_position_controller:
  #  joint: right_steering_hinge_joint
  #  type: effort_controllers/JointPositionController
#  pid: {p: 1.0, i: 0.0, d: 0.5}

  #pid: {p: 5.0, i: 0.0, d: 0.3}
  # rosrun rqt_reconfigure rqt_reconfigure parameter tuning